navbar-component.html

```diff
@@ -22,15 +24,16 @@
       </div>
 
       <!-- Collect the nav links, forms, and other content for toggling -->
-      <div class="collapse navbar-collapse" id="br-navbar-collapse">
+      <div ng-if="!$ctrl.service.brandOnly" class="collapse navbar-collapse"
+        id="br-navbar-collapse">
         <div br-slot="br-navbar-body">
           <div br-slot="br-navbar-menu">
             <!-- nav menu -->
             <ul class="nav navbar-nav">
               <li ng-repeat="menu in $ctrl.getDisplayedMenus()"
                 ng-class="{active: $root.route.vars.title == menu.pageTitle, dropdown: menu.items}">
                 <a ng-if="menu.visible && !menu.items"
-                  href="{{menu.url}}">
+                  href="{{menu.url}}" data-toggle="collapse" data-target=".navbar-collapse">
```

Comment:
Use `ng-click` in conjunction with `uib-collapse` and some `$ctrl.x` toggle var.